Sources/SwiftProtobuf/JSONDecodingOptions.swift

```diff
@@ -25,5 +25,9 @@ public struct JSONDecodingOptions {
   /// ignored, an error will be raised if one is encountered.
   public var ignoreUnknownFields: Bool = false
 
+  /// If unknown enum cases in the JSON should be defaulted to initial enum case.
+  /// If they aren't ignored, an error will be raised if one is encountered.
+  public var defaultUnknownEnumCases: Bool = false
```

Comment:
I think this should be called `ignoreUnkownEnumValues`

* This is parallel to `ignoreUnknownFields`

* In proto3, ignoring a field is the same as using the default value.

Reply:
Per other discussion, this has been merged into `ignoreUnknownFields`